prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+    h3{
+      text-align: center;
+    }
+    .container {
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+    }
+
+    .column {
+      flex-basis: 33%;
+      padding-left: 8px;
+      padding-right: 8px;
+      justify-content: center;
+      margin: 0 auto;
+    }
+
+  </style>
+</head>
+<body>
+
+  <h3>Indiana Jones</h3>
+
+  <div class="container">
+    <div class="row">
+      <div class="column">
+        <img src="images/raiders.jpg" alt="raiders image" style="width:100%"/>
+        <p>Raiders of the Lost Ark</p>
+        <p>I have never seen these movies but Indiana Jones is very cool.</p>
+
+        <div class="column">
+          <img src="images/temple.jpg" alt="temple image" style="width:100%"/>
+          <p>Temple of Doom</p>
+          <p>This film was released to financial success but inital reviews were mixed
+            critizing its darker elements and violence. However, critical opinion has improved
+            since 1984, citing the film's intensity and imagination.</p>
+
+          <div class="column">
+            <img src="images/crusade.jpg" alt="crusade image" style="width:100%"/>
+            <p>The Last crusade</p>
+            <p>After the mixed reaction to Indiana Jones and the Temple of Doom,
+              Speilberg chose to tone down the dark tone and graphic violence in the next
+              installment. During the five years between the Temple of the Doom and the
+              Last crusade, he and executive producer Lucas, reviewed several scripts before acccepting
+              Jeffrey Boam's.</p>
+
```

Comment:
To get the columns to appear next to each other, we'll want to close them before opening their next sibling.

```html
<div class="container">
  <div class="row">
    <div class="column">
    </div>
    <div class="column">
    </div>
    <div class="column">
    </div>
  </div>
</div>
```